tests/unit/api/cpp/merger/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# sACN C++ Merger API unit tests
+
+list(REMOVE_ITEM SACN_API_SOURCES ${SACN_SRC}/sacn/dmx_merger.c)
+list(APPEND SACN_MOCK_SOURCES ${SACN_SRC}/sacn_mock/dmx_merger.c)
```

Comment:
This is a fine approach, but if I need anything more granular than the full set of sources for a module in a unit test executable I tend to just list them individually instead of using the _SOURCES variables.